force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -93,17 +152,22 @@ public with sharing class DummyJSONCallout {
      */
 	
 	public static void postCreateDummyJSONUser(String contactId) {
```

Comment:
I am not sure this is being called from anywhere. Should be happening in the after update context whenever a DummyJSON_Id__c has changed